operations/security_and_authentication/Restrict_Access_to_NCN_Images_S3_Bucket.md

```diff
@@ -0,0 +1,687 @@
+# Procedure to Restrict Network Access to the ncn-images S3 Bucket
+
+The configuration documented in this procedure is intended to prevent user-facing dedicated nodes (UANs, Compute Nodes) for retrieving NCN image content from Ceph S3 Services, as running on storage nodes. Specifically, the controls enacted via this procedure should:
```

Comment:
```suggestion
The configuration documented in this procedure is intended to prevent user-facing dedicated nodes (UANs, Compute Nodes) from retrieving NCN image content from Ceph S3 services, as running on storage nodes. Specifically, the controls enacted via this procedure should:
```